reference/6/Microsoft.PowerShell.Core/About/about_Logging.md

```diff
@@ -0,0 +1,144 @@
+logging to the windows event log.
+
+### Filtering PowerShell log entries using rsyslog
+By default, PowerShell log entries are written to the default location/file
+for syslog.  However, it is possible to redirect the entries to a custom
+file.
+
+* Create a conf for PowerShell log configuration and provide a number that
+is less than 50 (for `50-default.conf`), such as `40-powershell.conf`. The
+file should be placed under `/etc/rsyslog.d`.
+
+* Add the following entry to the file
+```
+:syslogtag, contains, "powershell[" /var/log/powershell.log
+& stop
+```
+* Ensure `/etc/rsyslog.conf` includes the new file. Often, it will have
+a generic include statement that looks like following:
+
+`$IncludeCnofig /etc/rsyslog.d/*.conf`
+
+If it does not, you will need to add an include statement manually.
+
+* Ensure attributes and permissions are set appropriately
+
+```
+-rw-r--r-- 1 root root   67 Nov 28 12:51 40-powershell.conf
+```
+
+## Viewing PowerShell log output on MacOS
+
+The easiest method for viewing PowerShell log output on MacOS is using the Console application.
+
+* Search for the Console application and launch it
+* Select the Machine name under Devices
+* In the Search field, entry 'pwsh'; the PowerShell main binary.
+* Change search filter from 'Any' to 'Process'
+* Peform the operations
+* Optionally save the search for future use.
+
+To filter on a specific process instance of PowerShell in the Console, teh variable $pid contains the process id.
+* Enter the pid (Process Id) in the Search field.
+* Change search filter PID
+* Perform the operations
+
+### Viewing PowerShell log output from a command-line
+
+The `log` command can be used to view PowerShell log entries from the command-line.
+
+```
+sudo log stream --predicate 'process == "pwsh"' --info
+```
+
+## Configuring Logging on non-Windows systems
+On Windows, logging is configured by creating ETW trace listeners or by using the Event Viewer to enable Analytic logging.   On Linux and MacOS, logging isc onfigured using the file `PowerShellProperties.json`. The rest of this section will discuss configuring PowerShell logging on non-Windows system.
+
+By default, PowerShell enables informational logging to the operational channel.  What this means is any log output produced by PowerShell that is marked as operational and has a log (trace) level greater then informational will be logged.  Occasionally, diagnoses may require additional log output, such as verbose log output or enabling analytic log output.
+
+The file `PowerShellProperties.json` is a JSON formatted file residing in the PowerShell installation directory. Each installation of PowerShell uses it's own copy of this file. For normal operation, this file is left unchanged but it can be useful for diagnosis or for distinguishing between multiple PowerShell versions on the same system or even multiple copies of the same version (See LogIdentity in the table below).
```

Comment:
use `$PSHOME` instead of 'installation directory'

Reply:
fixed